packages/platform-ios/src/config/index.js

```diff
@@ -9,11 +9,14 @@
  */
 
 import path from 'path';
+import {memoize} from 'lodash';
 import findProject from './findProject';
 import findPodfilePath from './findPodfilePath';
 import findPodspec from './findPodspec';
 import type {UserConfigT} from 'types';
 
+const memoizedFindProject = memoize(findProject);
```

Comment:
`findProject` is expensive (runs FS operations through `glob`) and often called with the same `folder` argument, hence memoizing. Without this change, displaying the warning before builds would essentially be unusable, since it runs ~10s for as small as 4 native modules. With memoization the time is less than 800ms, yielding ~12x improvement that gets bigger with more deps.